### ISSUE TITLE
syntax error, unexpected '$conversion' (T_VARIABLE), expecting ',' or ')'

### ISSUE THREAD

**Issue by cjlaborde** [cpriego/valet-linux#274]
Having issue with https://github.com/spatie/laravel-medialibrary

I thought it was problem with spatie media.
https://i.imgur.com/pkqLNI9.png
Yet found out the first trace comes from valet-linux
https://i.imgur.com/y7aNDNs.png

Seems to be caused by 
```
require $frontControllerPath;
```
in `.composer/vendor/cpriego/valet-linux/server.php:232`

The app runs with with no error with PHP artisan serve

I fix it before my uninstalling and reinstalling valet-linux but error came again when I restart computer today.

Not sure if related but in composer had this issue.

```
Problem 1
    - phpspec/phpspec 5.1.2 requires php ^7.1, <7.4 -> your PHP version (7.4.3) does not satisfy that requirement.
    - phpspec/phpspec 5.1.2 requires php ^7.1, <7.4 -> your PHP version (7.4.3) does not satisfy that requirement.
    - phpspec/phpspec 5.1.2 requires php ^7.1, <7.4 -> your PHP version (7.4.3) does not satisfy that requirement.
    - Installation request for phpspec/phpspec (locked at 5.1.2, required as ^5.1) -> satisfiable by phpspec/phpspec[5.1.2].


```
Had to force it with `composer global update --ignore-platform-reqs `

**Comment by jmarcher** [comment]
It is not a Valet problem, you need PHP 7.4 for that code to work, check which version of PHP valet is using.

**Comment by cjlaborde** [comment]
@jmarcher  I am using PHP 7.4.3
as it is mentioned in Problem 1 section. Seems composer having a problem with the latest version of PHP
I fixed it by changing Valet PHP to use 7.2 then Setting it again to 7.4
Hopefully, the error doesn't happen again. So, for now, going close to this.
